src/indexation/pht.cpp

```diff
@@ -193,6 +193,14 @@ void Pht::lookupStep(Prefix p, std::shared_ptr<int> lo, std::shared_ptr<int> hi,
                 IndexEntry entry;
                 entry.unpackValue(*value);
 
+                auto it = std::find_if(vals->cbegin(), vals->cend(), [&](const std::shared_ptr<IndexEntry>& ie) {
+                    return ie->value == entry.value;
+                });
```

Comment:
Let's say, someone inexes a same value 1024 times. Then, the DHT node won't accept anymore values and at the PHT level, the nodes will refuse to enter the split process, therefor blocking the whole PHT, may be even from the root. I think you should not avoid splitting, but only filter values for the user, i.e. wrapping the `LookupCallback` passed by the user and filter the values there. The prefered place for doing so would be in the `Pht::lookup` function. 

@kaldoran, @aberaud